ui/oopsyraidsy/data/05-shb/alliance/the_puppets_bunker.js

```diff
@@ -0,0 +1,84 @@
+// TODO: 4FF4/4FF5 One of these is failing chemical conflagration
+// TODO: standing in wrong teleporter?? maybe 5363?
+
+[{
+  zoneId: ZoneId.ThePuppetsBunker,
+  damageWarn: {
+    'Puppet Aegis Beam Cannons 1': '5074', // rotating separating white ground aoe
+    'Puppet Aegis Beam Cannons 2': '5075', // rotating separating white ground aoe
+    'Puppet Aegis Beam Cannons 3': '5076', // rotating separating white ground aoe
+    'Puppet Aegis Collider Cannons': '507E', // rotating red ground aoe pinwheel
+    'Puppet Aegis Surface Laser 1': '5091', // chasing laser initial
+    'Puppet Aegis Surface Laser 2': '5092', // chasing laser chasing
+    'Puppet Aegis Flight Path': '508C', // blue line aoe from flying untargetable adds
+    'Puppet Aegis Refraction Cannons 1': '5081', // refraction cannons between wings
+    'Puppet Aegis Life\'s Last Song': '53B3', // ring aoe with gap
+    'Puppet Light Long-Barreled Laser': '5212', // line aoe from add
+    'Puppet Light Surface Missile Impact': '520F', // untargeted ground aoe from No Restrictions
+    'Puppet Superior Incendiary Bombing': '4FB9', // fire puddle initial
+    'Puppet Superior Sharp Turn': '506D', // sharp turn dash
+    'Puppet Superior Standard Surface Missile 1': '4FB1', // Lethal Revolution circles
+    'Puppet Superior Standard Surface Missile 2': '4FB2', // Lethal Revolution circles
+    'Puppet Superior Standard Surface Missile 3': '4FB3', // Lethal Revolution circles
+    'Puppet Superior Sliding Swipe 1': '506F', // right-handed sliding swipe
+    'Puppet Superior Sliding Swipe 2': '5070', // left-handed sliding swipe
+    'Puppet Superior Guided Missile': '4FB8', // ground aoe during Area Bombardment
+    'Puppet Superior High-Order Explosive Blast 1': '4FC0', // star aoe
+    'Puppet Superior High-Order Explosive Blast 2': '4FC1', // star aoe
+    'Puppet Heavy Energy Bombardment': '4FFC', // colored magic hammer-y ground aoe
+    'Puppet Heavy Revolving Laser': '5000', // get under laser
+    'Puppet Heavy Energy Bomb': '4FFA', // getting hit by ball during Active Suppressive Unit
+    'Puppet Heavy R010 Laser': '4FF0', // laser pod
+    'Puppet Heavy R030 Hammer': '4FF1', // circle aoe pod
+    'Puppet Hallway High-Powered Laser': '50B1', // long aoe in the hallway section
+    'Puppet Hallway Energy Bomb': '50B2', // running into a floating orb
+    'Puppet Compound Mechanical Dissection': '51B3', // spinning vertical laser
+    'Puppet Compound Mechanical Decapitation': '51B4', // get under laser
+    'Puppet Compound Mechnical Contusion Untargeted': '51B7', // untargeted ground aoe
+    'Puppet Compound 2P Relentless Spiral 1': '51AA', // triple untargeted ground aoes
+    'Puppet Compound 2P Relentless Spiral 2': '51CB', // triple untargeted ground aoes
+    'Puppet Compound 2P Prime Blade Out': '541F', // initial prime blade get out
+    'Puppet Compound 2P Prime Blade Out 2': '5198', // teleporting/reproduce prime blade get out
```

Comment:
Is one of these Compound 2P and the other Puppet 2P?

Reply:
Some of them are both.  I'll fix the comments.